src/booleans.js

```diff
@@ -1,62 +1,94 @@
 function negate(a) {
   // your code here
+  return !(a > 0);
 };
 
 function both(a, b) {
   // your code here
+  return a > 0 && b > 0;
 };
 
 function either(a, b) {
   // your code here
+  return a > 0 || b > 0;
 };
 
 function none(a, b) {
   // your code here
+  return !(a > 0 || b > 0);
 };
 
 function one(a, b) {
   // your code here
+  if (a > 0 && b > 0) {
+    return false;
+  }
+  return a > 0 || b > 0;
 };
+  
+
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
```

Comment:
There isnt any use in casting the `a` variable to a Boolean here, as it already is a boolean! Instead, we can use two NOT operators (!!) together to create a double negation. What this does is to first evaluate the value to a boolean that will be the opposite of what it actually is, eg passing in a string (which is a truthy value) will be made `false` by one of the ! operators, then the second ! will flip that to `true`, and so evaluating the string as being truthy. Nice trick that gets used more often than you might think!
```
  return !!a
```